dev-lang/ispc/ispc-1.16.0.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+
+inherit cmake toolchain-funcs python-any-r1 llvm
+
+LLVM_MAX_SLOT=12
+
+DESCRIPTION="Intel SPMD Program Compiler"
+HOMEPAGE="https://ispc.github.io/"
+
+if [[ ${PV} = *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/ispc/ispc.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
```

Comment:
`KEYWORDS=` shouldn't appear twice in an ebuild (notably so ekeyword tool is not confused), normally you shouldn't have the empty `=""` one and define keyboards only in the else block. This is how the live ebuild should be as well so these simple copies work.

Reply:
fixed, and moved it in live ebuild as well